app.py

```diff
@@ -10,6 +11,20 @@
     template_folder="./static/react",
 )
 
+
```

Comment:
Should we put this after the register blueprint or this is fine?

Reply:
I haven't tried shifting it around, but I believe this block of code has to, at the very least, come before the React routes are established (which all come before register blueprint, coincidentally).